pom.xml

```diff
@@ -103,7 +103,7 @@
 		<xformsVersion>4.3.12</xformsVersion>
 
 		<!-- core configuration library -->
-		<ugandaemrReportsVersion>2.0.17</ugandaemrReportsVersion>
+		<ugandaemrReportsVersion>2.0.17-SNAPSHOT</ugandaemrReportsVersion>
```

Comment:
@solemabrothers correct version is 2.0.18-SNAPSHOT

Reply:
Sorry just sent the wrong commit